CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Changed
 
+- Gateway EUI is no longer unset when deleting a gateway, meaning it could be recovered if no other gateway claimed it. This requires the `20220913000000_update_gtw_eui_index` migration to be executed in order to be functional.
```

Comment:
Look into how we referenced this in the past changelog entries:
`- This requires a database schema migration (`ttn-lw-stack is-db migrate`) because of added columns and views.`

The migration names are not visible to users, nor relevant.